src/chttpd/src/chttpd_db.erl

```diff
@@ -502,6 +502,19 @@ db_req(#httpd{method='POST',path_parts=[_,<<"_purge">>]}=Req, Db) ->
     Options = [{user_ctx, Req#httpd.user_ctx}, {w, W}],
     {IdsRevs} = chttpd:json_body_obj(Req),
     IdsRevs2 = [{Id, couch_doc:parse_revs(Revs)} || {Id, Revs} <- IdsRevs],
+    MaxIds = config:get_integer("purge", "max_document_id_number", 100),
+    case length(IdsRevs2) =< MaxIds of
+        false -> throw({bad_request, "Document numbers larger than expected"});
```

Comment:
The working is unclear. How about:
- `Too many documents in a single purge request`
- `The number of documents in a single purge requests is greater than the limit`
- `Exceeded maximum number of documents`

Reply:
Used `Exceeded maximum number of documents.`